### ISSUE TITLE
COCO val $mmap$

### ISSUE THREAD

**Issue by sxy1122** [ucas-vg/TOV_mmdetection#9]
Sorry, I don't know which precision this COCO val $mmap$ represents. Which of AP0.5, AP0.75, AP0.5-0.95, APs, APm and APl  is obtained by the network pre-training on the coco data set?

**Comment by yinglang** [comment]
It follows the common defination. mmAP is the mean iou of mean class of AP, so it means AP0.5-0.95.

**Comment by sxy1122** [comment]
> It follows the common defination. mmAP is the mean iou of mean class of AP, so it means AP0.5-0.95.

Thank you for your answer. I have another question. I ran it myself according to the code and instructions provided by you, but the result of pre training on coco was only 18.5, 1.1 less than 19.6 provided by you. Can you tell me why? Are there any changes to the specific settings?

**Comment by sxy1122** [comment]
![image](https://user-images.githubusercontent.com/77336413/135741207-906b9350-d5ad-40dd-b915-e2083f4edc06.png)

Also, does this refer to the range of various sizes mentioned in the paper? Then why is it different from what is said in the paper?

**Comment by yinglang** [comment]
> swer. I have another question. I ran it myself according to the code and instructions provided by you, but the result of pre training on coco was only 18.5, 1.1 less than 19.6 provided by you. Can you tell me why? Are there any changes to the specific settings?

Do you add val dataset of coco to train?
And if you use the trained weight as pretrian weight, the performance on TinyPerson is right. That's enough. Cause the performance on TinyCOCO is no meaning except for debug.

**Comment by yinglang** [comment]
> ![image](https://user-images.githubusercontent.com/77336413/135741207-906b9350-d5ad-40dd-b915-e2083f4edc06.png)
> 
> Also, does this refer to the range of various sizes mentioned in the paper? Then why is it different from what is said in the paper?

The output of log is area range, not size range, where area=size*size.

**Comment by sxy1122** [comment]
> > swer. I have another question. I ran it myself according to the code and instructions provided by you, but the result of pre training on coco was only 18.5, 1.1 less than 19.6 provided by you. Can you tell me why? Are there any changes to the specific settings?
> 
> Do you add val dataset of coco to train? And if you use the trained weight as pretrian weight, the performance on TinyPerson is right. That's enough. Cause the performance on TinyCOCO is no meaning except for debug.

I did not train Val dataset as a train dataset. I just downloaded the data set and used the code you provided. But I'm coco_ The precision of SM training is not up to what you provided, and then the effect of fine-tuning with tinyperson data set is a little worse than yours, so I wonder if the precision of fine-tuning is lower than 51.78 you provided because the pre training results are not achieved. Sorry, there are a lot of questions. Thank you for your patience.

**Comment by sxy1122** [comment]
> > ![image](https://user-images.githubusercontent.com/77336413/135741207-906b9350-d5ad-40dd-b915-e2083f4edc06.png)
> > Also, does this refer to the range of various sizes mentioned in the paper? Then why is it different from what is said in the paper?
> 
> The output of log is area range, not size range, where area=size*size.

thank you

**Comment by sxy1122** [comment]
![image](https://user-images.githubusercontent.com/77336413/135950575-54124b8a-d7de-4794-a231-686139674067.png)

Does this code indicate that the verification set and the test set use the same data set? So why do you use the same data set for testing and verification? Won't this cause accuracy error?

![image](https://user-images.githubusercontent.com/77336413/135950854-d5b9b9eb-e086-45a1-8b37-5337a47ad20f.png)

And where can I find the files in this data? I downloaded the data set of coco2017, but I don't have this file, only the file you commented out.

**Comment by yinglang** [comment]
Yes, TinyPerson Version 1(this publish version) have no test set because of there is no enough data. We will fixed the problem for following version which may take some time to publish.

For instances_merge2017.json, it is just simply from merging all box annotation of coco. Maybe you can refer to [this](https://github.com/ucas-vg/TinyBenchmark/tree/master/tiny_benchmark/configs/TinyCOCO#train-scale-match-coco-or-monotonous-scale-match-coco-for-tinyperson) for some detail. (The given instances_simple_merge2014.json is same as instances_merge2017.json)
